Story/HuntersMoon.cs

```diff
@@ -0,0 +1,111 @@
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreStory.cs
+using Skua.Core.Interfaces;
+
+public class HuntersMoon
+{
+    public IScriptInterface Bot => IScriptInterface.Instance;
+    public CoreBots Core => CoreBots.Instance;
+    public CoreStory Story = new();
+
+    public void ScriptMain(IScriptInterface bot)
+    {
+        Core.SetOptions();
+
+        StoryLine();
+
+        Core.SetOptions(false);
+    }
+
+    public void StoryLine()
+    {        
+        if (Core.isCompletedBefore(6810))
+            return;
+
+        Story.PreLoad(this);
+
+        #region Marchosias Fight
+        // Curse of the Moon 6699
+        Story.KillQuest(6699, "marchosiasfight", "Marchosias");
+        #endregion
+
+        #region Hunter's Moon
+        // Put out the Fire 6700
+        Story.KillQuest(6700, "huntersmoon", "Dark Fire");
+        
+        // The Most Dire of Beasts 6701
+        Story.KillQuest(6701, "huntersmoon", "Eclipsed One");
+        
+        // Find the Elder 6702
+        Story.MapItemQuest(6702, "huntersmoon", 1);
```

Comment:
You need to add MapitemId instead of 1 then quantity
Story.MapItemQuest(QuestId, MapName, MapItemId, Quantity)